_data/changelogs/range-slider.yml

```diff
@@ -19,4 +19,44 @@ changelogURL:
   #   githubPr:
   #   githubRepo:
   #   versionUswds:
-# end changelog item
\ No newline at end of file
+
+items:
+- date: 2022-04-28
+  summary: Released USWDS 3.0.
+  summaryAdditional: USWDS 3.0 allows teams to update their projects to modern Sass syntax, improve performance, and reduce the size of project CSS.
+  isBreaking: true
+  affectsAccessibility: true
+  affectsAssets: true
+  affectsGuidance: true
+  affectsJavascript: true
+  affectsMarkup: true
+  affectsStyles: true
+  githubPr: 4656
+  githubRepo: uswds
+  versionUswds: 3.0.0
+- date: 2022-04-11
+  summary: Added support for High Contrast Mode and forced colors
+  summaryAdditional: All our components now support proper display when users have a forced colors mode set in their operating system.
+  isBreaking: false
+  affectsAccessibility: true
+  affectsAssets: false
+  affectsGuidance: false
+  affectsJavascript: false
+  affectsMarkup: false
+  affectsStyles: true
+  githubPr: 4610
+  githubRepo: uswds
+  versionUswds: 2.13.3
+- date: 2021-08-18
+  summary: Added appropriate ARIA label to Range input
+  isBreaking: true
```

Comment:
I don't think this is a breaking change, but I am open to disagreement.

Reply:
I read Jame's note in https://github.com/uswds/uswds/pull/4270 about an additional enhancement being a breaking change ad mistook it for this update! I'll fix this now